README.md

```diff
@@ -1 +1,79 @@
-# solution-pack-malware-response-using-siem-and-edr-solutions
+# Malware Response Using SIEM & EDR Solutions
+
+## Release Information
+- Solution Pack Version: 1.0.0
+- Minimum Compatible FortiSOAR™ Version : 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
+## Overview
+### Introduction
+This article describes the Malware Response Using SIEM & EDR Solutions Pack. The solution pack demonstrates the scenarios and use cases around Malware reported by ElasticSearch or Splunk.
+
+Configure ingestion using ElasticSearch or Splunk connector. Ingestion process creates an alert of type 'Malware', and then manually triggers the response workflow.
```

Comment:
This can go  as a usage. config ingestion is a must have stp?

Reply:
As discussed with Abhishek it comes under introduction and we already mentioned all these connector config in SP "Instructions" 
Also, in all other SP we have mentioned config in intro

Reply:
k